test/persist/test_failure_indices.sh

```diff
@@ -0,0 +1,124 @@
+#! /bin/bash
+
+export PATH="$GOBIN:$PATH"
+export TMHOME=$HOME/.tendermint_persist
+
+rm -rf "$TMHOME"
+tendermint init
+
+# use a unix socket so we can remove it
+RPC_ADDR="$(pwd)/rpc.sock"
```

Comment:
If we're going to put stuff in `$HOME` anyway, we might as well do that here too.
```suggestion
readonly RPC_ADDR="$HOME/rpc.sock"
```